app/src/main/java/com/itachi1706/cheesecakeutilities/Modules/ConnectivityQuietHours/Receivers/NotificationHelper.java

```diff
@@ -0,0 +1,89 @@
+package com.itachi1706.cheesecakeutilities.Modules.ConnectivityQuietHours.Receivers;
+
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+
+import com.itachi1706.cheesecakeutilities.Modules.ConnectivityQuietHours.ConnectivityQuietHoursActivity;
+import com.itachi1706.cheesecakeutilities.Modules.ConnectivityQuietHours.QHConstants;
+import com.itachi1706.cheesecakeutilities.R;
+
+import java.text.DateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Random;
+
+import androidx.annotation.Nullable;
+import androidx.core.app.NotificationCompat;
+
+/**
+ * Created by Kenneth on 1/1/2019.
+ * for com.itachi1706.cheesecakeutilities.Modules.ConnectivityQuietHours in CheesecakeUtilities
+ */
+public class NotificationHelper {
+
+    private NotificationHelper() {
+        throw new IllegalStateException("Utility class. Do not instantiate like this");
+    }
+
+    public static ArrayList<String> lines = null;
```

Comment:
Remove all public reference as they can be package private. Also convert to Kotlin

Reply:
Done